File: test/fixtures/node/redirect_variable_and_string_data.js
import fetch from 'node-fetch';
import fs from 'fs';

fetch('http://localhost:28139', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: new ArrayBuffer('foo&', fs.readFileSync('start' + process.env['FILENAME'] + 'end', 'utf-8'))
});

File: test/fixtures/ruby-httparty/redirect_variable_and_string_data.rb
require 'httparty'

url = 'http://localhost:28139'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = 'foo&@start' + ENV['FILENAME'] + '$end'
res = HTTParty.post(url, headers: headers, body: body)
